lib/actions/form.js

```diff
@@ -37,12 +38,18 @@ export function setTime (payload) {
   }
 }
 
+let debouncedPlanTrip  // store as global, so it can be reused.  Maybe there's a better way?
```

Comment:
the debounce time is a variable that comes from the store, so it could change.  I updated my code so that it'll update the debounce function if that happens.